www/react-base/src/components/ForceBuildModal/Fields/FieldInt.tsx

```diff
@@ -34,7 +34,7 @@ export const FieldInt = observer(({field, fieldsState}: FieldIntProps) => {
       <div className="col-sm-10">
         <input data-bb-test-id={`force-field-${field.fullName}`}
                type="text" className="form-control" value={state.value}
```

Comment:
With the change to the state storage, maybe this should be change to `type="number"?

Reply:
Agreed